pkgconfig/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# CMakeLists.txt for libyui/pkgconfig
+#
+# Support for pkg-config:
+#
+# Generate a libyui.pc file from libyui.pc.in and install it to
+# /usr/lib64/pkgconfig.
+#
+# A .pc file specifies how to use a development package, in particular linker
+# flags (-lyui), compiler flags (including include directories) and paths.
+# See man pkg-config.
+#
+# Notice that libyui this is the ONLY libyui* package where a .pc file makes
```

Comment:
The sentence sounds weird to me: "Notice that libyui this is the ONLY libyui* ...".

Reply:
Right now, we have 9 libyui* repos, each of which generates one binary package and most additionally one -devel and one -doc package.